src/server.js

```diff
@@ -48,6 +48,9 @@ app.use((req, res, next) => {
 
 // Global variables
 
+// Routes
+app.use("/teams", require("./routes/teams.routes"));
```

Comment:
Please, try only use this file app.use("/", require("./routes/at-resources.routes"));